### PR TITLE
I want to pull request

### PR DESCRIPTION
**Pull request by FrancisLydz1321**
TESTiNG

### CODE REVIEW

**Comment by Daweee** [comment]
Task linked: [CU-86cw1utzf FE - Driver Assignment and Scheduling](https://app.clickup.com/t/86cw1utzf)

File: tripnitor_mobile_app/lib/main.dart [hunk 1]
```diff
@@ -31,95 +31,8 @@ class MyApp extends StatelessWidget {
         colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
         useMaterial3: true,
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
+      home: Text('HELL WOLRD'),
```
Comment (Daweee) [inline tripnitor_mobile_app/lib/main.dart:34]: still lacking screens -test

**Comment by Daweee** [comment]
not final, need revision -test

File: tripnitor_mobile_app/lib/main.dart (hunk 1 above)
Comment (FrancisLydz1321) [reply tripnitor_mobile_app/lib/main.dart:34]: hi boss

**Pull request closed without merging**